cmd/applyconfig/applyconfig.go

```diff
@@ -239,6 +242,55 @@ func applyConfig(rootDir, cfgType string, process processFn) error {
 	return nil
 }
 
+type censoringFormatter struct {
```

Comment:
Why can't we import this from `test-infra`?

Reply:
In `test-inra`, the censoring logger is associated with an agent, which I do not need/have in the context of `applyConfig`.